### ISSUE TITLE
Return whole csv response as strings or use proper float?

### ISSUE THREAD

**Issue by FabiKo117** [GIScience/ohsome-api#170]
While fixing issue #166 I saw that within the csv response, all the float values are returned as strings. Do we want to keep it like that?

So which of these two responses do you think is better:

1)
```csv
# Copyright URL: https://ohsome.org/copyrights
# Copyright Text: © OpenStreetMap contributors
# API Version: 1.4.1
timestamp;value;value2;ratio
"2016-01-01T00:00:00Z";"5643.0";"1178.0";"0.208754"
"2017-01-01T00:00:00Z";"6015.0";"1100.0";"0.182876"
```

2)
```csv
# Copyright URL: https://ohsome.org/copyrights
# Copyright Text: © OpenStreetMap contributors
# API Version: 1.5.0-SNAPSHOT
timestamp;value;value2;ratio
2016-01-01T00:00:00Z;5643.0;1178.0;0.208754
2017-01-01T00:00:00Z;6015.0;1100.0;0.182876
```

There is an optional boolean value `applyQuotesToAll` in the `writeNext()` method of the used csv library that is used to write each line, which could be set accordingly, meaning it would not be possible to just use it for the numbers, but not for the date for example. 
See here the mentioned function in the ohsome API code:
https://github.com/GIScience/ohsome-api/blob/352e961db8225c4405aaa3a19b9777fb4ae9496b/src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java#L334

Anyway my preference is on `2)` because numbers should not really be represented as strings if possible. I'm not sure if we've already had a conversation over that in the past or why we went for the string representation here.

**Comment by bonaparten** [comment]
Said that we should probably use commas instead of semicolons in the CSV files, if we are talking only about float values, I think we shouldn't use the double-quotes ([ietf.org](https://www.ietf.org/rfc/rfc4180.txt)).

**Comment by FabiKo117** [comment]
I've checked in our archived ohsome API instance in GitLab on issues regarding the CSV integration. In fact, both of the here mentioned things, the double quotes, as well as the semicolon as delimiter have their reason.

Regarding the semicolon as delimiter: quote: `I've moved to using the semicolon as delimiter, as the request URL includes the comma, so that there are no problems. I will also add a check on a semicolon for the custom ids in /groupBy/boundary to avoid troubles.`

Regarding double quotes: In OSM, there are tags like `landuse=meadow;farmland`and to make sure that the semicolon does not get used as a delimiter, it has to be under double quotes.

So actually this already has been thought through and therefore I will close this issue again.
